python/setup.py

```diff
@@ -241,7 +241,7 @@ def get_option_from_sys_argv(option_str):
     os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION'] = 'cpp'
 
   # Keep this list of dependencies in sync with tox.ini.
-  install_requires = ['six>=1.9', 'setuptools']
+  install_requires = ['six>=1.9']
```

Comment:
Per the comment, can you verify whether any change to `tox.ini` is required?

Reply:
Funnily enough - tox.ini was NOT in sync with this file, in the one way that it was missing setuptools :).